tur/python-seledroid/build.sh

```diff
@@ -0,0 +1,14 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/luanon404/seledroid
+TERMUX_PKG_DESCRIPTION="Python library to control android webdriver."
+TERMUX_PKG_LICENSE="Unlicense"
+TERMUX_PKG_MAINTAINER="@nacho00112"
+TERMUX_PKG_VERSION=1.1.0
+TERMUX_PKG_SRCURL=https://github.com/luanon404/seledroid/archive/refs/heads/main.zip
```

Comment:
Do not use `main.zip`. This is not a proper way when packaging a package. If any change is pushed to the main branch and then the checksum will change too.